src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -108,7 +108,6 @@ private async Task MvcTemplateCore(string languageOverride)
         [InlineData(true)]
         [InlineData(false)]
         [SkipOnHelix("cert failure", Queues = "OSX.1014.Amd64;OSX.1014.Amd64.Open")]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19716")]
```

Comment:
Linked issue has been closed with a fix that we believe was affecting this test.